app/services/record_declaration.rb

```diff
@@ -20,13 +20,7 @@ class RecordDeclaration
   validates :declaration_date, presence: { message: I18n.t(:missing_declaration_date) }
   validates :declaration_type, presence: { message: I18n.t(:missing_declaration_type) }
   validates :declaration_date, future_date: true, declaration_date: true, allow_blank: true
-  validates :evidence_held,
-            presence: { message: I18n.t(:missing_evidence_held), if: :validate_evidence_held? },
-            inclusion: {
-              in: ->(record_declaration) { record_declaration.participant_profile.class::VALID_EVIDENCE_HELD },
-              if: :validate_evidence_held?,
-              message: I18n.t(:invalid_evidence_type),
-            }
+  validates :evidence_held, evidence_held: true
```

Comment:
could we move this down as the last validation to line 30 to avoid it returning with other validations incase?